src/tracing/agent.h

```diff
@@ -8,7 +8,14 @@
 namespace node {
 namespace tracing {
 
-using v8::platform::tracing::TracingController;
+class TracingController : public v8::platform::tracing::TracingController {
+ public:
+  TracingController() : v8::platform::tracing::TracingController() {}
+
+  int64_t CurrentTimestampMicroseconds() override {
+    return uv_hrtime();
```

Comment:
is it OK to return nanoseconds instead of microseconds?

Reply:
Pretty sure it is okay. But properly we should use microseconds, such that it is less of a  breaking change.

Reply:
@targos thanks for catching this! I had a division in my original test commit but I ended up losing while manually porting the change. D'oh.

@AndreasMadsen The trace-event framework [requires](https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/edit#) the timestamp to be in micro-seconds, and all the current tooling available in chrome or through the [catapult project ](https://github.com/catapult-project/catapult) would not work correctly if this was in nano-seconds.